CMakeLists.txt

```diff
@@ -8,7 +8,7 @@
 
 # Preamble ####################################################################
 #
-cmake_minimum_required(VERSION 3.18.0)
+cmake_minimum_required(VERSION 3.24.0)
```

Comment:
Also all the documentations needs to be updated to say cmake 3.24 instead of 3.18

Reply:
Updated :+1: